complete_test.go

```diff
@@ -214,6 +223,42 @@ func TestComplete(t *testing.T) {
 	}
 }
 
+// ExampleComplete_outputCapturing demonstrates the ability to capture
+// the output of Complete() invocations, crucial for integration tests.
+func ExampleComplete_outputCapturing() {
+	// a stringLookup function maps one string to another.
```

Comment:
Maybe for the sake of making the example easier to read, define them outside the function?

Reply:
I did consider it, but then fell back to this, as I didn't think it was worth littering the package scope with symbols that have no other use but this one silly test. But, sure, can do.

Reply:
Thanks
Looks much cleaner to my eyes :eyes:

Reply:
I'm glad I could contribute an improvement.

Could you also, please, mark as such the conversations that you feel are resolved?